default/species.txt

```diff
@@ -649,6 +669,8 @@ Species
         [[HAS_ADVANCED_FOCI]]
     ]
     
+    preferredfocus = "FOCUS_GROWTH"
```

Comment:
Oh, good call, well done, I approve of that choice, didn't think of Growth as an option for any of them.